lego/apps/polls/fixtures/development_polls.yaml

```diff
@@ -13,20 +14,41 @@
 - model: polls.Option
   pk: 1
   fields:
-    name: Skra
-    votes: 3
+    name: Tinder
+    votes: 5
     poll: 1
 
 - model: polls.Option
   pk: 2
   fields:
-    name: Bombom
-    votes: 25
+    name: Grinder
```

Comment:
offside 😨

Reply:
So you're saying that **Grinder** is offside, but **Tinder** is okey?  I dont think Webkom should discriminate like this 🔥

Reply:
Very true, I apologize to the Offside community.

Reply:
Typo tho: https://www.grindr.com/

Reply:
🕵